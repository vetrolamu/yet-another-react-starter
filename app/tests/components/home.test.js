import React from 'react';
import Header from '../../components/header/header.jsx';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <Header title="Hello" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
