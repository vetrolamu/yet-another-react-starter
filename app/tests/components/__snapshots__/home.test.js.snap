exports[`test renders correctly 1`] = `
<h1
  title="Hello">
  Hello
</h1>
`;
